src/components/app/DocumentList.js

```diff
@@ -969,7 +969,8 @@ export class DocumentList extends Component {
               tabIndex={0}
               indentSupported={layout.supportTree}
               disableOnClickOutside={clickOutsideLock}
-              limitOnClickOutside={isIncluded && isModal}
+              // limitOnClickOutside={isIncluded && isModal}
+              limitOnClickOutside={isModal}
```

Comment:
@siemiatj: Not sure if this is the correct way, but it looks like prop. isIncluded is undefined for DocumentList when used inside a RawModal. This happens because  in Container component, we use <DocumentList> at line 154 with no isIncluded prop. Is this correct?

Reply:
It is set when there are two grids in a RawModal. But maybe this `isIncluded` can be skipped here, or we can create another case ?

![Screen Shot 2019-05-21 at 17 20 18](https://user-images.githubusercontent.com/513460/58108857-dce4ef00-7bec-11e9-89ae-132e444bdfb2.png)

Did a quick recording so that you can check this use case http://recordit.co/QXHEyQdV6I

Reply:
Even for this case the solution to have `limitOnClickOutside={isModal}` is working. So I just removed the commented  old version and this should be ready to merge.